Release_ISOs

```diff
@@ -16,10 +16,10 @@ fi
 # Constants
 ROOTDIR="$PWD"
 OUTPUTDIR="output-MinGW-i386"
-BOOTCDISO="ReactOS-${version}.iso"
-BOOTCDZIP="ReactOS-${version}-iso.zip"
-LIVECDISO="ReactOS-${version}-Live.iso"
-LIVECDZIP="ReactOS-${version}-live.zip"
+BOOTCDISO="ReactOS-${version}-${ROS_ARCH}-iso"
```

Comment:
Are you sure, that the dot should become a minus here? Looks wrong to me and inconsitent to LIVECDISO where you kept the dot.

Reply:
That is not correct indeed.

Reply:
Resolved.